spyder/plugins/variableexplorer/widgets/dataframeeditor.py

```diff
@@ -250,8 +262,9 @@ def max_min_col_update(self):
             # the maximum of a column.
             # Fixes spyder-ide/spyder#17145
             try:
-                if col.dtype in REAL_NUMBER_TYPES + COMPLEX_NUMBER_TYPES:
-                    if col.dtype in REAL_NUMBER_TYPES:
+                if (is_any_real_numeric_dtype(col.dtype)
+                        or col.dtype in COMPLEX_NUMBER_TYPES):
```

Comment:
```suggestion
                if (
                    is_any_real_numeric_dtype(col.dtype)
                    or col.dtype in COMPLEX_NUMBER_TYPES
                ):
```

Small formatting improvement.